src/org/jitsi/impl/neomedia/transform/dtls/DatagramTransportImpl.java

```diff
@@ -482,7 +482,15 @@ public void send(byte[] buf, int off, int len)
             switch (type)
             {
             case ContentType.handshake:
-                short msg_type = TlsUtils.readUint8(buf, off + 11);
+                short msg_type
+                    = TlsUtils.readUint8(
+                            buf,
+                            off
+                                + 1 /* type */
+                                + 2 /* version */
+                                + 2 /* epoch */
+                                + 6 /* sequence_number */
+                                + 2 /* length */);
```

Comment:
Maybe we should print a warning when we encounter an unknown message type (the "default" in the switch statement below)?